nafas/functions.py

```diff
@@ -206,6 +208,62 @@ def input_filter(input_data):
     return filtered_data
 
 
+def load_config(config_path):
+    """
+    Load configuration from a JSON file.
+
+    :param config_path: config path
+    :type config_path: str
+    :return: result as dict
+    """
+    try:
+        with open(config_path, 'r') as config_file:
+            config_data = json.load(config_file)
+        if not validate_config(config_data):
+            raise Exception
+        program_data = {
+            'ratio': [
+                config_data['ratio']['inhale'],
+                config_data['ratio']['retain'],
+                config_data['ratio']['exhale'],
+                config_data['ratio']['sustain']
+            ],
+            'unit': config_data['unit'],
+            'pre': config_data['pre'],
+            'cycle': config_data['cycle'],
+        }
+        return {
+            "status": True,
+            "data": {
+                "program_name": config_data['name'],
+                "program_level": "Custom",
+                "program_data": program_data}}
```

Comment:
```
                "program_data": program_data
            }
        }
```

Reply:
fixed in https://github.com/sepandhaghighi/nafas/pull/129/commits/675be1de32af00c53c469332b3a29886f6192024